### DISCUSSION TITLE
Can we display a top % instead of rank in user GitHub stats?

### DISCUSSION THREAD

**Discussion by LincolnKermit** [anuraghazra/github-readme-stats#3934] [Q&A] [answered]
Hello there,
Is it possible to display a top % instead of rank in user GitHub stats?
For example : 1% instead of S Tier Rank.
Lincoln.

**Comment by lulunac27a** [comment, marked as answer]
Yes, it's possible by setting `rank_icon` to `percentile` like `&rank_icon=percentile`. See [Stats Card Exclusive Options
](https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options) for more information.

**Reply by qwerty541** [reply]
That's correct. Thank you for answering.

I will also add an example there:

```html
<img src="https://github-readme-stats.vercel.app/api?username=lincolnkermit&show_icons=true&include_all_commits=true&theme=dracula&locale=en&rank_icon=percentile" />
```

<img src="https://github-readme-stats.vercel.app/api?username=lincolnkermit&show_icons=true&include_all_commits=true&theme=dracula&locale=en&rank_icon=percentile" />

**Comment by LincolnKermit** [comment]
Thanks!
